frontend/src/views/Crops/Crops.tsx

```diff
@@ -30,6 +30,10 @@ function Crops() {
   const [editingCropIndex, setEditingCropIndex] = useState<number | undefined>(undefined);
   const [isDialogOpen, setIsDialogOpen] = useState<boolean>(false);
 
+  const [showDeleteDialog, setShowDeleteDialog] = useState<boolean>(false);
```

Comment:
Sorry, I didn't explicitly ask these other booleans to be removed, but I do think it's cleaner to use the dialogText.